vector/src/main/java/im/vector/app/features/MainActivity.kt

```diff
@@ -142,9 +140,9 @@ class MainActivity : VectorBaseActivity<ActivityMainBinding>(), UnlockedActivity
         startAppViewModel.onEach {
             renderState(it)
         }
-        startAppViewModel.viewEvents.stream()
-                .onEach(::handleViewEvents)
-                .launchIn(lifecycleScope)
+        startAppViewModel.observeViewEvents(Lifecycle.State.CREATED) {
```

Comment:
Need to use CREATED here, or the app does not start (race condition?)

Reply:
It's really weird no? What exactly do you mean by doesn't start?

Reply:
The app stays in the MainActivity forever, the event `AppStarted` is sent but not received.